data/meson.build

```diff
@@ -25,8 +25,9 @@ if get_option('tests')
 endif
 
 if build_standalone
-  install_data(['daemon.conf'],
-    install_dir: join_paths(sysconfdir, 'fwupd')
+  install_data(['fwupd.conf'],
+    install_dir: join_paths(sysconfdir, 'fwupd'),
+    install_mode: 'rw-r-----',
```

Comment:
Crazy thought - if we're putting everything in a single conffile, do we need an `/etc/fwupd` anymore?  I guess there is `remotes.d`, but we could just as well have remotes as part of `fwupd.conf` too...

Reply:
> but we could just as well have remotes as part of fwupd.conf too

I think that would upset a lot of users -- quite a few people "drop in a file" to add a remote and I think that's super useful.